addons/website/static/src/js/menu/edit.js

```diff
@@ -63,18 +63,28 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         url.searchParams.delete('enable_editor');
         url.searchParams.delete('with_loader');
         window.history.replaceState({}, null, url);
+
+        if (this._editorAutoStart) {
+            $('body').block({overlayCSS: {
+                backgroundColor: '#000',
+                opacity: 0.5,
+                zIndex: 1050
+            }, message: false});
+        }
     },
     /**
      * Auto-starts the editor if necessary or add the welcome message otherwise.
      *
      * @override
      */
-    start: function () {
+    start: async function () {
```

Comment:
Minor: Since you edit this line
```suggestion
    async start() {
```